charts/federation-v2/README.md

```diff
@@ -29,24 +29,37 @@ group](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster).
 
 ## Installing the Chart
 
-First you'll need to create the reserved namespace for registering clusters with the
-cluster registry:
+First you'll need to create the reserved namespace for registering clusters with the cluster registry:
 
 ```bash
 $ kubectl create ns kube-multicluster-public
 ```
 
-Install the chart with the release name `federation-v2`:
+Next, add the chart to your local resitory.
+```bash
+$ helm repo add federation-v2 https://raw.githubusercontent.com/kubernetes-sigs/federation-v2/master/charts/federation-v2
+
+$ helm repo list
+NAME            URL                                        
+federation-v2   https://raw.githubusercontent.com/kubernetes-sigs/federation-v2/master/charts/federation-v2
+```
+
+With the chart installed, availble versions can be viewed.
+```bash
+$ helm search federation-v2
+```
 
+Install the chart and append the version you would like using the `--version` tag. Here, we specify version 0.0.7 and name it federation-v2.
+Note, the chart will default to 0.0.7.
```

Comment:
The chart should not default to 0.0.7.  If there is a default, it should be to the latest released version.

Reply:
Removed default note.